src/QuadForm/LineOrbits.jl

```diff
@@ -54,16 +54,30 @@ function Base.show(io::IO, P::LineEnumCtx)
   end
 end
 
-Base.length(P::LineEnumCtx) = P.length
+Base.length(P::LineEnumCtx) = BigInt(P.length)
```

Comment:
You said you cannot change this because otherwise `collect` won't work. But if `P.length` truly does not fit into an `Int`, then `collect` won't work anyway because you just tried to use more RAM than humanity has created in total in the course of its existence.

At the same time, you slow down the common case where it *does* fit.

You could just do this:
```suggestion
Base.length(P::LineEnumCtx) = Int(P.length)
```
That should work for any "small" value of length, and will otherwise give an error. Of course if any code wants the "true" length then it needs to get it directly or use a different accessor.

Reply:
As far as I have seen, the `length` itself is only use in a `_line_orbits` code where we create a vector of length `length`, which should fit in what you say. I guess replacing by `Int` should be fine for that case since it was working in that way before. Thank you for pointing that out.